### DISCUSSION TITLE
Asking for verification

### DISCUSSION THREAD

**Discussion by SKAZEXE** [community/community#65376] [GitHub Education]
I applied and verified my profile of the student developer pack 2 years ago for 4 years but now after 2 years github again asking me to verify again and they couldn't verify it! Any solution?

**Comment by queenofcorgis** [comment]
Welcome to the GitHub Community, @SKAZEXE , we're happy you're here! You are more likely to get a useful response if you are posting your question in the applicable category. I've gone ahead and moved it for you. Good luck!

**Reply by SKAZEXE** [reply]
I've been using it for the past two years, and its total duration is four years. However, they are now requesting verification again after only half of the expected time has passed.

**Comment by sksaleha** [comment]
Is there any chances to reject of my student developer pack as my current status is: Congrats! Your request for @sksaleha was approved and your benefits will be available soon.
Could you please help me?

**Reply by SKAZEXE** [reply]
they will be available soon.

**Comment by sksaleha** [comment]
And I applied on 27th of these month and when will I get the benefits?

**Comment by Sachin066** [comment]
they will be available soon.
